main.go

```diff
@@ -82,6 +82,672 @@ var (
 	masterMetrics = []snapshotMetric{
 		snapshotMetric{
 			desc: prometheus.NewDesc(
+				"mesos_master_cpus_percent",
+				"Percent of CPUs used as a fraction of 1.",
```

Comment:
What do you mean by fixing? I'm still not sure if we try to keep metric names as close to mesos names as possible.